pkgs/applications/misc/freeplane/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchurl, unzip, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  version = "1.8.11";
+  pname = "freeplane";
+
+  src = fetchurl {
```

Comment:
```suggestion
  src = fetchzip {
```

Reply:
fetchzip fails with the problem described in https://github.com/NixOS/nixpkgs/issues/60157:

```
[...]
building '/nix/store/39v4zj0xy5w7xcx4p16zxb3hsdlwfn2q-freeplane_bin-1.8.11.zip.drv'...
note: keeping build directory '/tmp/nix-build-freeplane_bin-1.8.11.zip.drv-0'
error: --- Error --- nix-daemon
builder for '/nix/store/39v4zj0xy5w7xcx4p16zxb3hsdlwfn2q-freeplane_bin-1.8.11.zip.drv' failed with exit code 1; last 9 log lines:

  trying https://sourceforge.net/projects/freeplane/files/freeplane%20stable/freeplane_bin-1.8.11.zip/download
    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                   Dload  Upload   Total   Spent    Left  Speed
  100   467  100   467    0     0    668      0 --:--:-- --:--:-- --:--:--   667
  100   381  100   381    0     0    275      0  0:00:01  0:00:01 --:--:--  2140
  100 41.5M  100 41.5M    0     0  16.9M      0  0:00:02  0:00:02 --:--:-- 79.4M
  unpacking source archive /build/download
  do not know how to unpack source archive /build/download
```

Reply:
The funny workaround in https://github.com/NixOS/nixpkgs/issues/60157#issuecomment-524965720 might work ...